tests/fail/external_C/function_not_in_SO.stderr

```diff
@@ -0,0 +1,14 @@
+error: unsupported operation: can't call foreign function: foo
+  --> $DIR/function_not_in_SO.rs:LL:CC
+   |
+LL |         foo();
+   |         ^^^^^ can't call foreign function: foo
+   |
+   = help: this is likely not a bug in the program; it indicates that the program performed an operation that the interpreter does not support
```

Comment:
oh, we could extend this diagnostic to also suggest the new ffi flag

Reply:
good idea!